assets/stylesheets/sections/_notifications.scss

```diff
@@ -8,6 +8,7 @@
 		right: 0;
 		bottom: 0;
 	min-width: 400px;
+	z-index: z-index( 'root', '#wpnt-notes-panel2' );
```

Comment:
Parent stacking context is #header (which is the masterbar) instead of 'root' so it should be:

```
z-index: z-index( '.masterbar', '#wpnt-notes-panel2' );
```

Where:
```
	'.masterbar': (
                '#wpnt-notes-panel2': yourvalue,
		'.masterbar__notifications-bubble': 99999,
		'.reader-back': 99999,
		'.reader-profile': 100000,
		'.reader-visit-site': 99999
	),
```